### PR TITLE
Update with the Polymer 1.1 style recommendations

### PR DESCRIPTION
**Pull request by abdonrd**

### CODE REVIEW

**Comment by samccone** [comment]
lgtm :+1: thx a bunch @abdonrd :star:

**Comment by abdonrd** [comment]
Your welcome @samccone! :smiley:

**Comment by abdonrd** [comment]
Re-updated to master! :smiley:

File: bower.json [hunk 1]
```diff
@@ -20,16 +20,15 @@
...
+    "polymer": "Polymer/polymer#^1.1.0"
   },
   "devDependencies": {
-    "web-component-tester": "*",
     "iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
+    "paper-styles": "PolymerElements/paper-styles#^1.0.0",
     "test-fixture": "PolymerElements/test-fixture#^1.0.0",
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0",
-    "paper-styles": "PolymerElements/paper-styles#^1.0.0"
+    "web-component-tester": "*",
```
Comment (samccone) [inline bower.json:32]: why the change?

File: paper-header-panel.html [hunk 2]
```diff
@@ -135,256 +135,240 @@
...
+          }
+        },
+
+        _scroll: function() {
+          this._keepScrollingState();
+          this.fire('content-scroll', {target: this.scroller}, {bubbles: false});
+        },
+
+        _getScrollerForMode: function(mode) {
+          return MODE_CONFIGS.outerScroll[mode] ? this : this.$.mainContainer;
```
Comment (samccone) [inline paper-header-panel.html]: why the formatting change here

File: bower.json (hunk 1 above)
Comment (abdonrd) [inline bower.json:32]: When I update a version, bower sort alphabetically.

**Comment by samccone** [comment]
@abdonrd just a future note, if you would not mind when you make these PRs only change the style pattern and not the formatting or unrelated package version info. When you make unrelated changes it makes these diffs much harder to review as well as goes against the principle of atomic commits http://www.freshconsulting.com/atomic-commits/

File: paper-header-panel.html (hunk 2 above)
Comment (abdonrd) [inline paper-header-panel.html]: You mean to put the script into the dom-module?
I do this because the documentation examples of Polymer 1.1 are made this way.

**Comment by abdonrd** [comment]
@samccone okey!
I used to making atomic commits, but Monica told me to do a commit by PR.
I think that if the `bower.json` was always sort alphabetically we would avoid problems.

**Comment by samccone** [comment]
> Monica told me to do a commit by PR. 

:+1: 1 atomic commit :)

If you would like to fixup the styles and sorting (which I am :+1: for) Just open up another PR :)

**Comment by abdonrd** [comment]
Perfect! I take note! Thank you @samccone!

File: paper-header-panel.html (hunk 2 above)
Comment (samccone) [inline paper-header-panel.html]: oh no you changed this line from being a mutli line ternary to a single line one.

Comment (abdonrd) [inline paper-header-panel.html]: Ah, okey! I will revert now.

**Comment by abdonrd** [comment]
Anything else, @samccone? :smiley:

**Comment by abdonrd** [comment]
Anything else, @samccone? :smiley:

**Comment by samccone** [comment]
nope this looks great @abdonrd 

cc @notwaldorf

**Comment by abdonrd** [comment]
Rebased to master! :star:

File: bower.json [hunk 3]
```diff
@@ -20,16 +20,15 @@
   "private": true,
   "license": "http://polymer.github.io/LICENSE.txt",
   "homepage": "https://github.com/PolymerElements/paper-header-panel",
-  "ignore": [],
```
Comment (notwaldorf) [inline bower.json:23]: why remove the ignore?

Comment (abdonrd) [inline bower.json:23]: @notwaldorf because it was empty. I restore it?

Comment (notwaldorf) [inline bower.json:23]: Yeaaaaaah, we seem to get bower warnings like `"invalid-meta iron-component-page is missing "ignore" entry in bower.json"` if we don't have them, so please add it back :)

**Comment by blasten** [comment]
Looking at this PR, I have a comment. The css file for header-panel is very small that I'd rather keep it in the same `.html` file. For 1.1, the style tag should now be inside the template. See https://github.com/PolymerElements/paper-slider/blob/master/paper-slider.html#L68 for reference. What do you think?

File: bower.json (hunk 3 above)
Comment (abdonrd) [inline bower.json:23]: Received!

Comment (abdonrd) [inline bower.json:23]: And updated!

**Comment by abdonrd** [comment]
@blasten yes, the css file for header-panel is small. I will move it now inside the `paper-header-panel.html`.

**Comment by abdonrd** [comment]
@notwaldorf @blasten updated! :muscle:

**Comment by blasten** [comment]
LGTM. Thanks!
